### DISCUSSION TITLE
Can this be used to control multiple physical boards?

### DISCUSSION THREAD

**Discussion by martman0823-2** [sflems/nhl-led-scoreboard-webgui#21] [Q&A]
Hi there,

Just wondering if this can be used to control multiple physical boards?  Say I have 5 different scoreboards with separate raspberry pi controllers.  Can all be controlled within this GUI?

**Comment by sflems** [comment]
In its current state, it can only manage one board at a time. I'll be sure to add this to the TODO list though!
